data/strings/strings.txt

```diff
@@ -26894,3 +26894,390 @@
+    cs = S vašimi dary a podporou můžeme vytvořit ty nejlepší mapy na světě!
+    da = Med dine donationer og støtte kan vi skabe de bedste kort i verden!
+    de = Mit Ihren Spenden und Ihrer Unterstützung können wir die besten Karten der Welt erstellen!
+    el = Με τις δωρεές και την υποστήριξή σας, μπορούμε να δημιουργήσουμε τους καλύτερους χάρτες στον κόσμο!
+    es = ¡Con sus donaciones y apoyo, podemos crear los mejores mapas del mundo!
+    et = Teie annetuste ja toetusega saame luua maailma parimaid kaarte!
+    eu = Zure dohaintza eta laguntzarekin, Munduko mapa onenak sor ditzakegu!
+    fa = با کمک های مالی و حمایت شما، ما می توانیم بهترین نقشه های جهان را ایجاد کنیم!
+    fi = Lahjoituksellasi ja tuellasi voimme luoda maailman parhaat kartat!
+    fr = Avec vos dons et votre soutien, nous pouvons créer les meilleures cartes du monde !
+    he = עם התרומות והתמיכה שלך, נוכל ליצור את המפות הטובות בעולם!
+    hi = आपके दान और समर्थन से, हम दुनिया में सबसे अच्छे मानचित्र बना सकते हैं!
+    hu = Adományaival és támogatásával létrehozhatjuk a világ legjobb térképes alkalmazását!
+    id = Dengan donasi dan dukungan Anda, kami dapat membuat peta terbaik di Dunia!
+    it = Con le tue donazioni e il tuo supporto, possiamo creare le migliori mappe del mondo!
+    ja = あなたの寄付とサポートがあれば、世界で最高の地図を作成できます。
+    ko = 여러분의 기부와 지원으로 우리는 세계 최고의 지도를 만들 수 있습니다!
+    mr = तुमच्या देणग्या आणि समर्थनासह, आम्ही जगातील सर्वोत्तम नकाशे तयार करू शकतो!
+    nb = Med dine donasjoner og støtte kan vi lage de beste kartene i verden!
+    nl = Met uw donaties en steun kunnen we de beste kaarten ter wereld maken!
+    pl = Dzięki Twoim darowiznom i wsparciu możemy tworzyć najlepsze mapy na świecie!
+    pt = Com suas doações e apoio, podemos criar o melhor mapa do mundo!
+    pt-BR = Com suas doações e apoio, podemos criar o melhor mapa do mundo!
+    ro = Cu donațiile și sprijinul dumneavoastră, putem crea cele mai bune hărți din lume!
+    ru = Благодаря вашим пожертвованиям и поддержке мы сможем создать лучшие карты на свете!
+    sk = S vašimi darmi a podporou môžeme vytvoriť tie najlepšie mapy na svete!
+    sv = Med dina donationer och stöd kan vi skapa de bästa kartorna i världen!
+    sw = Kwa michango na usaidizi wako, tunaweza kuunda ramani bora zaidi Duniani!
+    th = ด้วยการบริจาคและการสนับสนุนของคุณ เราสามารถสร้างแผนที่ที่ดีที่สุดในโลกได้!
+    tr = Bağışlarınız ve desteğinizle dünyanın en iyi haritalarını yaratabiliriz!
+    uk = Завдяки вашим пожертвам і підтримці ми зможемо створити найкращі карти на світі!
+    vi = Với sự đóng góp và hỗ trợ của bạn, chúng tôi có thể tạo ra những bản đồ tốt nhất trên Thế giới!
+    zh-Hans = 有了您的捐赠和支持，我们可以创建世界上最好的地图！
+    zh-Hant = 有了您的捐贈和支持，我們可以創建世界上最好的地圖！
+
+  [app_tip_02]
+    comment = App tip #02
+    tags = android
+    en = Do you like our app? Please donate to support the development! Don't like it yet? Please let us know, and we will fix it!
```

Comment:
```suggestion
    en = Do you like our app? Please donate to support the development! Don't like it yet? Please let us know why, and we will fix it!
```
That changes the sense a bit.

Reply:
Wording modified... I will request to the different contributors to review once again the translation of this tip...